pkgs/by-name/bi/binwalk/package.nix

```
@@ -110,6 +96,31 @@ rustPlatform.buildRustPackage rec {
   doInstallCheck = true;
   versionCheckProgramArg = "-V";
 
+  postInstall = ''
+    wrapProgram $out/bin/binwalk --prefix PATH : ${
```

```suggestion
    wrapProgram $out/bin/binwalk --suffix PATH : ${
```

This makes it easier to override these packages